css/customUi.css

```diff
@@ -38,3 +38,8 @@ form.usertext {
 #header {
     display: none;
 }
+
+div.pref-row {
+    border-top: 1px dotted gray;
```

Comment:
I assume the correct color would be var(--md-sys-color-outline-variant), per https://m3.material.io/components/divider/specs